src/NuGet.Server/Web.config

```diff
@@ -43,7 +43,13 @@
     <!--
     Set enableFrameworkFiltering to true to enable filtering packages by their supported frameworks during search.
     -->
-    <add key="enableFrameworkFiltering" value="false"/>
+    <add key="enableFrameworkFiltering" value="false" />
+    
+	<!--
+	If value is setted to 0, there are no retention otherwise the retention will be setted to the defined value
+	-->
+	<add key="retentionRule" value="0" />
```

Comment:
nit: Alignment for top 4 lines. And comment: `If the value is set to 0, there is no retention otherwise the retention will be set to the defined value`